python/orca/example/learn/pytorch/super_resolution/super_resolution.py

```diff
@@ -90,7 +90,7 @@ def download_bsd300(dest=opt.data_dir):
     output_image_dir = join(dest, "BSDS300/images")
 
     if not exists(output_image_dir):
-        makedirs(dest)
+        makedirs(dest, exist_ok=True)
```

Comment:
what does this change do?

Reply:
In case that ./dataset already exists; otherwise, an error would arise.